README.md

```diff
@@ -1,6 +1,8 @@
+[![Coverage Status](https://coveralls.io/repos/github/windranger-io/windranger-solidity-template/badge.svg?branch=master)](https://coveralls.io/github/windranger-io/windranger-solidity-template?branch=master)
```

Comment:
Do we need to create the windranger-io organisation on Coveralls I wonder? The docs for the Coveralls workflow plugin says the repo (and thus badge) will be created automatically from the plugin.

How do we prevent a downstream template repo from using this (windranger-solidity-template) badge link automatically?

Reply:
>How do we prevent a downstream template repo from using this (windranger-solidity-template) badge link automatically?

Unless authentication tokens are involved, I don't think we can prevent that 🤷